CITATION.cff

```diff
@@ -0,0 +1,7 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+title: "Triton Inference Server: An Optimized Cloud and Edge Inferencing Solution."
```

Comment:
Where does this line come from?

Reply:
This is our official "About" description on the github repo

![image](https://user-images.githubusercontent.com/21284872/227375855-9759b9d2-4c35-44cf-90ac-0eab464c5247.png)